contrib/scalar/scalar.txt

```diff
@@ -129,6 +130,14 @@ With the `--all` option, all enlistments currently registered with Scalar
 will be reconfigured. This option is meant to to be run every time Scalar
 was upgraded.
 
+Delete
+~~~~~~
+
+delete <enlistment>::
+    This command lets you delete an existing Scalar enlistment from your
+    local file system, unregistering the repository and stopping any file
+    watcher daemons (FSMonitor).
```

Comment:
Remove this bit about FS Monitor currently and discuss it as a potential future option in the commit message.

Reply:
Done.